docs/docs/91-migrations.md

```diff
@@ -8,6 +8,9 @@ Some versions need some changes to the server configuration or the pipeline conf
 - Removed `WOODPECKER_ROOT_PATH` and `WOODPECKER_ROOT_URL` config variables. Use `WOODPECKER_HOST` with a path instead
 - Pipelines without a config file will now be skipped instead of failing
 - Deprecated `includes` and `excludes` support from **event** filter
+- Deprecated alternative names for secrets, use `environment` with `from_secret`
+- Deprecated slice definition for labels and env vars
+- Deprecated `detached` in favor of services
```

Comment:
I dont think we should depricate detatched. There is no equivalent ... services wont do as they start before while detatched can start and use e.g. artifacts build by a step before